acceptance/cmd/fs/cp/dir-to-dir/test.toml

```diff
@@ -0,0 +1,20 @@
+Local = true
+Cloud = false
```

Comment:
I recommend setting Cloud = true for this test as well. That validates the implementation against a real server. You can use `fs cat` to confirm that the files were uploaded.